scripts/codegen_swagger_models.py

```diff
@@ -34,15 +35,31 @@
 GENERATED_MODEL_ROOT = "client/swagger_client/models"
 GENERATED_MODEL_IMPORT_PATH = "swagger_client.models"
 
-MODEL_ROOT_CAST = "skema/program_analysis/CAST2GrFN/model/cast"
-IMPORT_PATH_CAST = "skema.program_analysis.CAST2GrFN.model.cast"
+MODEL_ROOT_CAST = "skema/program_analysis/CAST2FN/model/cast"
+IMPORT_PATH_CAST = "skema.program_analysis.CAST2FN.model.cast"
 
 MODEL_ROOT_GROMET = "skema/gromet/fn"
 IMPORT_PATH_GROMET = "skema.gromet.fn"
 
 MODEL_ROOT_METADATA = "skema/gromet/metadata"
 IMPORT_PATH_METADATA = "skema.gromet.metadata"
 
+GROMET_VERSION_CANONICAL_DEFINITION = "skema/gromet/__init__.py"
+
+
+# -----------------------------------------------------------------------------
+# Update Canonical GROMET_VERSION
+# -----------------------------------------------------------------------------
+
+def update_canonical_gromet_version(new_version):
+    path = os.path.join('..', GROMET_VERSION_CANONICAL_DEFINITION)
+    with open(path, 'r') as gv:
+        file_contents = re.sub(r"\s*GROMET_VERSION\s*=\s+\"[\d.]+\"",
```

Comment:
I think we should change the regex to `r"\s*GROMET_VERSION\s*=\s+\".*?$"`

Reply:
Done!